src/libs/modulegroups.c

```diff
@@ -37,8 +37,17 @@ typedef struct dt_lib_modulegroups_t
 {
   uint32_t current;
   GtkWidget *buttons[DT_MODULEGROUP_SIZE];
+  GtkWidget *text_entry;
+  GtkWidget *hbox_buttons;
+  GtkWidget *hbox_search_box;
 } dt_lib_modulegroups_t;
 
+typedef enum dt_lib_modulegroup_iop_visibility_type_t {
```

Comment:
sorry again this :)

{ should be at start of line

Reply:
I don't know what to do about this, I use clang and is doing it like this. clang is the recommended way to go, so it should be modified to format it the way you like it. Otherwise I'll have to disable for this and enable for the rest.

Reply:
You mean that clang compiler is changing that? Don't know I'm using emacs and gcc? But I do compile with clang to check PR and don't have this issue.